### PR TITLE
Fix multifrustum seam artifacts

### PR DESCRIPTION
**Pull request by lilleyse**
Fixes https://github.com/AnalyticalGraphicsInc/cesium/issues/4381

I decided not to go with the stencil approach mentioned in that issue because it would have gotten very complicated with derived commands.

The new idea is to keep two framebuffers, one for the globe and one for everything else. After all frustums have been rendered the "everything-else" framebuffer is overlayed above the globe framebuffer.

The new framebuffer is only created and used if `depthTestAgainstTerrain = false` and the number of frustums is greater than 1.

https://github.com/AnalyticalGraphicsInc/cesium/pull/8202 needs to be merged first. I also want to do more testing.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @lilleyse!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/fix-multifrustum/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by lilleyse** [comment]
I checked most of the Sandcastle examples with log-depth and depth-test-against-terrain set to false and the only one that had an issue was the VR example. I pushed a fix for that.

@IanLilleyT ready to review

**Comment by lilleyse** [comment]
You might notice this seam in the Sandcastle demo. I opened a separate issue for that https://github.com/AnalyticalGraphicsInc/cesium/issues/8216.

Selection_3067

**Comment by lilleyse** [comment]
Since we're getting a little close to the release let's target this PR, https://github.com/AnalyticalGraphicsInc/cesium/pull/8203, and https://github.com/AnalyticalGraphicsInc/cesium/pull/8207 for after the 1.62 release.

**Comment by lilleyse** [comment]
@IanLilleyT this is ready to review

File: Source/Scene/GlobeDepth.js [hunk 1]
```diff
@@ -39,6 +42,7 @@ define([
         DepthViewPacked,
```
Comment (IanLilleyT) [inline Source/Scene/GlobeDepth.js]: DepthViewPacked unused in this file

File: Source/Scene/GlobeDepth.js [hunk 2]
```diff
@@ -17,13 +18,15 @@ define([
         '../Shaders/PostProcessStages/DepthViewPacked',
```
Comment (IanLilleyT) [inline Source/Scene/GlobeDepth.js]: DepthViewPacked unused in this file

Comment (IanLilleyT) [inline Source/Scene/GlobeDepth.js]:
```suggestion
```

File: Source/Scene/GlobeDepth.js (hunk 1 above)
Comment (IanLilleyT) [inline Source/Scene/GlobeDepth.js]:
```suggestion
```

**Comment by lilleyse** [comment]
@IanLilleyT updated

**Comment by IanLilleyT** [comment]
Looks good, thanks @lilleyse

**Pull request merged**